src/challenges.js

```diff
@@ -1,54 +1,140 @@
 // Desafio 1
-function compareTrue() {
-  // seu código aqui
+function compareTrue(value1, value2) {
+  if (value1 === true && value2 === true) {
+    return true;
+  }
+  return false;
 }
```

Comment:
Boa noite Lucio, muito bom seu projeto! O que você acha de alterar essa função para este formato?
```suggestion
function compareTrue(value1, value2) {
  return (value1 && value2) ;
}
```
Essa sugestão também se aplica a outras funções que possuem uma operação simples, e então podem ser realizadas no próprio retorno da função. Show seu projeto #VQV